internal/vsp/feepayment.go

```diff
@@ -731,7 +736,29 @@ func (fp *feePayment) reconcilePayment() error {
 	if feeTx == nil || len(feeTx.TxOut) == 0 {
 		err := fp.makeFeeTx(nil)
 		if err != nil {
-			fp.schedule("reconcile payment", fp.reconcilePayment)
+			var apiErr *BadRequestError
+			if errors.As(err, &apiErr) {
+				if apiErr.Code == codeTicketCannotVote {
```

Comment:
we can drop an indent by combining these if statements:

`if errors.As(err, &apiErr) && apiErr.Code == codeTicketCannotVote`